Library/Formula/nload.rb

```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
```

Comment:
sorry,  the last think i was trying to do is to force removal of this tool. It works mostly fine, is tiny and very useful (IMHO).